server/backend/api/app/crud/crud_user_role.py

```diff
@@ -258,57 +297,10 @@ def find_or_create_forest_client_child_role(
     return child_role
 
 
-def find_by_id(
-    db: Session, user_role_xref_id: int
-) -> models.FamUserRoleXref:
+def find_by_id(db: Session, user_role_xref_id: int) -> models.FamUserRoleXref:
     user_role = (
         db.query(models.FamUserRoleXref)
-        .filter(
-            models.FamUserRoleXref.user_role_xref_id == user_role_xref_id
-        )
+        .filter(models.FamUserRoleXref.user_role_xref_id == user_role_xref_id)
         .one_or_none()
     )
     return user_role
-
-
-class UserRoleValidator:
```

Comment:
This was very early code. The initial intention was to have a central validator to check for all business validations for user_role_assignment create request, not just for "forest_client_number". It was not in the validator folder initially and for some reason the role_assignment creation didn't use it for other validations anymore... 
I think it is ok that you create a new validator in the validator folder only for forest_client_number.

Reply:
yeah but if we check the code, it's actually just checking the forest client numbers... I feel it's okay to remove it